sky/tools/create_ios_framework.py

```diff
@@ -33,27 +36,31 @@ def main():
   simulator_dylib = os.path.join(simulator_framework, 'Flutter')
 
   if not os.path.isdir(arm64_framework):
-    print 'Cannot find iOS arm64 Framework at', arm64_framework
+    print('Cannot find iOS arm64 Framework at %s' % arm64_framework)
```

Comment:
Why the change to the prints?

Reply:
Move them to python3 friendly syntax.  I'm assuming at some point the bots will be on python3